src/graphql/public/schema.graphql

```diff
@@ -79,6 +80,16 @@ type AccountUpdateDisplayCurrencyPayload {
   errors: [Error!]!
 }
 
+input AccountUpdatePushNotificationSettingsInput {
+  disabledPushNotificationTypes: [PushNotificationType]!
```

Comment:
typically we also want to make the inner mandatory (with another !)

the difference is that:
with the outer, the argument must exist, with the inner, the argument must be an array. 

if you just had the inner, it would say that if the argument exist, it must be an array, but it could be undefined also. 

```suggestion
  disabledPushNotificationTypes: [PushNotificationType!]!
```

Reply:
Will make this change.